test/exercises_test.dart

```diff
@@ -112,19 +112,19 @@ Running tests for: $packageName
 }
 
 void main() {
-  final testname = Platform.environment[ENV_NAME];
+  final testName = Platform.environment[envName];
 
   test("Exercises", () async {
-    if (testname == null) {
+    if (testName == null) {
       await runAllTests();
     } else {
-      final testpath = "${Directory.current.path}/exercises/$testname";
+      final testPath = "${Directory.current.path}/exercises/$testName";
 
-      if (!await new Directory(testpath).exists()) {
-        throw new ArgumentError("No exercise with this name: $testname");
+      if (!await new Directory(testPath).exists()) {
+        throw new ArgumentError("No exercise with this name: $testName");
       }
 
-      await runTest(testpath);
+      await runTest(testPath);
```

Comment:
These changes were made in a separate PR that was already merged into master, no need to include it here.